src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets

```diff
@@ -65,4 +67,36 @@ Copyright (c) .NET Foundation. All rights reserved.
       Include="$(MSBuildThisFileDirectory)..\analyzers\Microsoft.CodeAnalysis.NetAnalyzers.dll"
       IsImplicitlyDefined="true" />
   </ItemGroup>
+
+  <!-- C# Code Style Analyzers -->
+  <ItemGroup Condition="$(EnableNETAnalyzers) And '$(Language)' == 'C#'">
```

Comment:
perhaps I need to define a different property here. This would mean installing FxCop analyzers will disable codestyle analysis

Reply:
Yes, a different property sounds better. Given that there are no warning analyzers in this package (and should never be one in future), should that property be defaulted to true for all managed projects regardless of the TFM?

Reply:
I would like to keep analyzer behavior for other frameworks the same as previous releases. I do not want to have to test and support a giant matrix of frameworks

Reply:
That is good point. Thanks!